mpas_analysis/default.cfg

```diff
@@ -1578,7 +1577,7 @@ seasons =  ['ANN']
 comparisonGrids = ['latlon', 'subpolar_north_atlantic']
 
 # list of tuples(pairs) of depths (min, max) to integrate horizontal transport over
-depthRanges = [(0.0, -10000.0), (0.0, -2000.0)]
+depthRanges = [(10.0, -10000.0), (10.0, -2000.0)]
```

Comment:
It turns out that zero is not a safe upper bound because this cuts off a few parts of the ocean where the SSH is more than 1 m high (where the middle of the 2-m top layer can be above zero).